<launch>
  <arg name="ns" default="morai"/>
  <group ns="$(arg ns)">
    <!-- 퍼셉션: 창 끄기 -->
    <node pkg="roscpp_morai" type="lane_kante_ver2_morai.py"
          name="lane_perception" output="screen" respawn="true">
      <param name="show_window" value="true"/>
    </node>

    <node pkg="roscpp_morai" type="control_center_morai.py"
          name="lane_controller" output="screen" respawn="true"/>
  </group>
</launch>
